test/core/utils/preload-cssom.js

```diff
@@ -0,0 +1,169 @@
+describe('axe.utils.preloadCssom', function() {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+	var pageTpl =
+		'<!DOCTYPE html><html><head><style>html {width: 100vh;}</style></head><body></body></html>';
+	var args;
+
+	beforeEach(function() {
+		args = {
+			asset: 'cssom',
+			timeout: 10000
+		};
+	});
+
+	afterEach(function() {
+		fixture.innerHTML = '';
+	});
+
+	it('should be a function', function() {
+		assert.isFunction(axe.utils.preloadCssom);
+	});
+
+	it('should return a queue', function() {
+		fixture.innerHTML = pageTpl;
```

Comment:
The fixture is a `div` in the test page. That's not something you can just innerHTML a page into and expect it to work.